go.mod

```diff
@@ -30,42 +30,42 @@ require (
 	google.golang.org/genproto v0.0.0-20201211151036-40ec1c210f7a // indirect
 	google.golang.org/grpc v1.34.0
 	gopkg.in/yaml.v2 v2.4.0 // indirect
-	k8s.io/api v0.20.0
-	k8s.io/apimachinery v0.20.0
-	k8s.io/client-go v1.20.0
-	k8s.io/component-base v0.20.0
+	k8s.io/api v1.21.0
```

Comment:
Please take a close look at the original line.  It was v0.20.0, so it should be changed to v0.21.0.
If it was v1.20.0, then change to v1.21.0.

This comment applies to all changes.